frontend/packages/console-app/src/components/file-upload/file-upload-utils.ts

```diff
@@ -0,0 +1,24 @@
+import { ResolvedExtension } from '@console/dynamic-plugin-sdk/src/api/useResolvedExtensions';
```

Comment:
there is already a file with same name here `packages/dev-console/src/components/jar-file-upload/jar-file-upload-utils.ts`

Reply:
`file-upload-utils` is for generic file upload
`jar-file-upload-utils` if specifically for jar files